### PR TITLE
deprecation fixes for Julia 0.5

### PR DESCRIPTION
**Pull request by mlubin**
@tkelman, does this seem reasonable?

### CODE REVIEW

File: src/grb_model.jl [hunk 1]
```diff
@@ -104,7 +106,7 @@ end
 
 # read / write file
 
-function read_model(model::Model, filename::ASCIIString)
+function read_model(model::Model, filename::String)
```
Comment (tkelman) [inline src/grb_model.jl]: do utf8 filenames work even on windows?

**Comment by tkelman** [comment]
I usually prefer something more descriptive than assert, but mostly

**Comment by mlubin** [comment]
> I usually prefer something more descriptive than assert, but mostly

I'll leave it to Gurobi to make the interface more professional when they feel like it

File: src/grb_model.jl (hunk 1 above)
Comment (mlubin) [inline src/grb_model.jl]: No idea, but non-ascii filenames should work in principle

Comment (tkelman) [inline src/grb_model.jl]: the win32 unicode api's will usually be in utf16 though, so it depends what gurobi uses to manipulate files

Comment (mlubin) [inline src/grb_model.jl]: I have no motivation to spend time figuring this out and testing on different platforms. Should I add an ``isascii`` assert with a TODO?

Comment (tkelman) [inline src/grb_model.jl]: I think that would be safer

**Pull request merged**